layouts/partials/header.html

```diff
@@ -46,6 +46,10 @@
                                 {{ end }}
                             </div>
                         </div>
+                        <div class="navbar-search results-hidden">
+                            <input id="search_mobile" class="search" placeholder="Search" autocomplete="off">
+                            <div id="search_results_mobile"></div>
+                        </div>
```

Comment:
This is also within the range loop on line 39, I think it should be placed outside as in the Antora UI.

Reply:
No.But I have placed it outside the as I have placed in Antora UI.
Actually I couldn't find till which line the range of the loop is in Hugo.

Reply:
Odd that you say that it's not within the loop then. The range loop starts on line 39 and ends on line 56